image_tools/src/showimage.cpp

```diff
@@ -81,9 +99,8 @@ int main(int argc, char * argv[])
   bool show_camera = true;
 
   // Configure demo parameters with command line options.
-  if (!parse_command_options(argc, argv, &depth, &reliability_policy, &history_policy,
-    &show_camera))
-  {
+  if (!parse_command_options(argc, argv, &depth, &reliability_policy,
+                             &history_policy, &show_camera)) {
```

Comment:
we usually wrap lines right after the opening parenthesis rather than between parameters (c.f. [Developer guide](https://github.com/ros2/ros2/wiki/Developer-Guide#c-1))